client/src/randomBag.ts

```diff
@@ -0,0 +1,36 @@
+import { TetrominoType } from "common/TetrominoType";
+
+export class RandomBag{
+    bagStack: Array<TetrominoType>;
+
+    constructor() {
+        this.bagStack = [];
+
+    }
+    
+    //using the Fisher Yates algorithm
+    private shuffle() {
+    let currIdx = this.bagStack.length - 1;
+
+    while (currIdx > 0) {
+        const randIdx = Math.floor(Math.random() * currIdx);
+        [this.bagStack[currIdx], this.bagStack[randIdx]] = [this.bagStack[randIdx], this.bagStack[currIdx]];
+        currIdx--;
+    }
+    return this.bagStack;
```

Comment:
Can you do a `pnpm install` in the project root folder? It will add the linter to auto format things on commit, so that code doesn't need to be formatted manually. see https://github.com/ECE493G14Capstone/Capstone/pull/55#issue-1150021530